### DISCUSSION TITLE
Timer/Duration feature

### DISCUSSION THREAD

**Discussion by xenonCBU** [angularsen/UnitsNet#1055] [Q&A] [answered]
Hello ! 

Do you know if already exist a functionality that works like a timer/stopwatch or a countdown timer?

**Comment by tmilnthorp** [comment, marked as answer]
Not in UnitsNet, no.

Check out [System.Timers.Timer](https://docs.microsoft.com/en-us/dotnet/api/system.timers.timer.elapsed?view=netstandard-2.0) and [System.Diagnostics.Stopwatch](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch?view=netstandard-2.0).

**Reply by xenonCBU** [reply]
Ok, thank you.
